packages/widget-chat/src/loader.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/no-set-state */
+import React, {Component} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles/loader.css';
+
+function getDisplayName(C) {
+  return C.displayName || C.name || `C`;
+}
+
+export default function injectWidgetLoader(WrappedComponent) {
+  class WidgetLoader extends Component {
+
+    constructor(props) {
+      super(props);
+      this.handleSubmit = this.handleSubmit.bind(this);
+      this.handleFieldChange = this.handleFieldChange.bind(this);
+    }
+
+    shouldComponentUpdate() {
+      return true;
+    }
+
+    getComponentProps(component) {
+      return component.propTypes;
+    }
+
+    handleFieldChange(e) {
+      this.setState({tempUserId: e.target.value});
+    }
+
+    handleSubmit(e) {
+      e.preventDefault();
+      this.setState(Object.assign({}, {
+        userId: this.state.tempUserId
+      }));
+    }
+
+    render() {
+      let widget;
+      const props = this.props;
+      if (this.state && this.state.userId) {
+        widget = ( // eslint-disable-line no-extra-parens
```

Comment:
Is there a way to get this eslint error fixed? Annoying when trying to assign multi-line react jsx to a var.